en-US/win10/samples/speechtranslator.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Blinky Sample
+permalink: /en-US/win10/samples/speechtranslator.htm
+lang: en-US
+---
+
+#Build the SpeechTranslator Project
+
+###Component Lists:
+
+- 2 Rpi2 boards
+- 2 Rpi2 Power Supplies 
+- 2 [Headsets](https://www.microsoft.com/hardware/en-us/p/lifechat-lx-3000/JUG-00013) 
+- 2 Mouses 
+- 2 Ethernet cables 
+- 2 SD cards 
+- 2 HDMI monitors 
+- 2 HDMI cable  
+- 1 Router 
+- 1 SD card reader
+
+
+### Setup your hardware
+- Follow the [instruction](http://ms-iot.github.io/content/en-US/win10/SetupRPI.htm) to flush your sd cards
+- Insert the sd card and USB headsets into devices; Also Connect HDMI monitors and Ethernet cable to the devices; You can connect both Ethernet cable to the router, then the router will be connected to the Internet
+- Once all connected, power the devices on
+- You should see the device name and device ip on the screen
+- Telnet/powershell into the device, 
+  type: `setcomputername speechtransrpi2` to rename one device, and type `setcomputername speechtransrpi1` to rename the other device;
+
+	Note: you can choose the other device name as well, but in constantParam.cs file(you will see this file once you download the sample and open the solution as below steps show), you need to match them up.
+	You will see the details later.
+
+- Restart the device: shutdown /r /t 0
+
+### Setup your sample:
+
+
+1. Download the sample from [here](https://github.com/ms-iot/samples/speechtranslator) to your local PC
+2. Open the solution file in visual studio
+3. Open the constantParam.cs file, you need to specify your own clientid and clientsecret
+
+	Note: this is an azure account which you need to access the MS translation service in azure;
+	Follow this [link](http://www.microsoft.com/en-us/translator/getstarted.aspx) to get a account, 
+	And	once you get the account, replace the clientid and clientsecret with yours.

```

Comment:
Can you provide an obfuscated screen shot of where to find it on the Microsoft account as well as that same data in the cs file?